tests/conftest.py

```diff
@@ -0,0 +1,259 @@
+import sys
```

Comment:
Like above, imports would be more readable in order

Reply:
fixed